test/acceptance_with_go_client/go.mod

```diff
@@ -0,0 +1,38 @@
+module acceptance_tests_with_client
+
+go 1.19
+
+require (
+	github.com/stretchr/testify v1.8.0
+	github.com/weaviate/weaviate-go-client/v4 v4.6.1
```

Comment:
Can you use here this version of go client:
```
github.com/weaviate/weaviate-go-client/v4 v4.6.3-0.20230307175645-1e6af74af18b
```
This version pulls newest contextionary dependency that's migrated to weaviate organization